k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -89,25 +89,25 @@
 import security.planck.mdm.RestrictionsReceiver;
 import security.planck.mdm.UserProfile;
 import security.planck.network.ConnectionMonitor;
+import security.planck.passphrase.PassphraseRepository;
 import security.planck.provisioning.ProvisioningManager;
 import security.planck.sync.KeySyncCleaner;
 import security.planck.sync.SyncRepository;
-import security.planck.ui.passphrase.PassphraseActivity;
-import security.planck.ui.passphrase.PassphraseRequirementType;
+import security.planck.ui.passphrase.old.PassphraseActivity;
+import security.planck.ui.passphrase.old.PassphraseRequirementType;
 import timber.log.Timber;
 import timber.log.Timber.DebugTree;
 
 @HiltAndroidApp
 public class K9 extends MultiDexApplication implements DefaultLifecycleObserver {
     public static final boolean DEFAULT_COLORIZE_MISSING_CONTACT_PICTURE = false;
     public PlanckProvider planckProvider;
-    private Account currentAccount;
-    private ConnectionMonitor connectivityMonitor = new ConnectionMonitor();
+    private final ConnectionMonitor connectivityMonitor = new ConnectionMonitor();
     private static boolean enableEchoProtocol = false;
     private static Set<MediaKey> mediaKeys;
     private Boolean runningOnWorkProfile;
 
-    private AtomicBoolean deviceJustLeftGroup = new AtomicBoolean(false);
+    private final AtomicBoolean deviceJustLeftGroup = new AtomicBoolean(false);
```

Comment:
why do we make this final? Avoid reinitialization of the boolean?

Reply:
The AtomicBoolean itself is never re-assigned, just its internal value.

Reply:
ok